drivers/mm/mm_drv_intel_adsp_mtl_tlb.c

```diff
@@ -62,15 +62,26 @@ DEVICE_MMIO_TOPLEVEL_STATIC(tlb_regs, DT_DRV_INST(0));
 	((ROUND_DOWN((hpsram_ebb_quantity) + 31u, 32u) / 32u) - 1u)
 
 #define L2_SRAM_PAGES_NUM			(L2_SRAM_SIZE / CONFIG_MM_DRV_PAGE_SIZE)
-#define MAX_EBB_BANKS_IN_SEGMENT	32
-#define SRAM_BANK_SIZE				(128 * 1024)
+#define MAX_EBB_BANKS_IN_SEGMENT		32
+#define SRAM_BANK_SIZE				KB(128)
 #define L2_SRAM_BANK_NUM			(L2_SRAM_SIZE / SRAM_BANK_SIZE)
-#define IS_BIT_SET(value, idx)		((value) & (1 << (idx)))
+#define IS_BIT_SET(value, idx)			((value) & (1 << (idx)))
+#define PMC_COMMUNICATION			defined(CONFIG_SOC_SERIES_INTEL_ACE) && \
+						defined(CONFIG_MISC_INTEL_COMM_WIDGET)
```

Comment:
Put "depends on SOC_SERIES_INTEL_ACE" into the Kconfig definition for "MISC_INTEL_COMM_WIDGET" and you don't need this extra term in the test, and can just use the raw kconfig directly instead of inventing your own flag.